src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -28,7 +28,7 @@ void SimulatedTestFixture::SetUp()
     // of hack because we're changing a global variable, but it can't be easily fixed
     // through dependency injection until
     // https://github.com/UBC-Thunderbots/Software/issues/1299
-    MutableDynamicParameters->init();
+    // MutableDynamicParameters->init();
```

Comment:
why is this just commented out and not deleted altogether?

Reply:
ah really good catch! uncommented